src/Nix/Options/Parser.hs

```diff
@@ -27,140 +29,163 @@ nixOptions :: UTCTime -> Parser Options
 nixOptions current =
   Options
     <$> (fromMaybe Informational <$> optional
-          (option
-            (do
-              a <- str
-              if all isDigit a
-                then pure $ decodeVerbosity (read a)
-                else fail "Argument to -v/--verbose must be a number"
-            )
-            (short 'v' <> long "verbose" <> help "Verbose output")
+        (option
```

Comment:
I think the loss of indentation makes the nesting less clear here, but, again, I don't care too much.